cmd/registry/push/push.go

```diff
@@ -192,6 +197,13 @@ func (o *pushOptions) runPush(ctx context.Context, args []string) error {
 		return err
 	}
 
+	if o.AutoFloatingTags {
+		t := strings.Split(o.Version, ".")
```

Comment:
Could we validate the version and ensure that it is in semver format? Hint: https://github.com/falcosecurity/falcoctl/blob/da9865e2096f992446e243a72c78ebd1d77ed96d/internal/follower/follower.go#L352-L355